web_client/js/views/visualization.js

```diff
@@ -58,27 +58,47 @@ histomicstk.views.Visualization = girder.View.extend({
             }, this));
         });
 
-        this.listenTo(histomicstk.events, 'query:image', _.bind(function (image) {
+        this.listenTo(histomicstk.events, 'query:image', function (image) {
             var currentImage = this._controlModel.get('value') || {};
             if (image && currentImage.id !== image) {
                 this._controlModel.set('value', new girder.models.ItemModel({_id: image}));
             } else if (!image) {
                 this.removeItem();
                 this._controlModel.set('value', null);
             }
-        }, this));
+        });
+
+        // Set image bounds on URL query parameter change
+        this.listenTo(histomicstk.events, 'query:bounds', this._boundsFromQuery);
 
         // fallback to canvas renderer rather than dom
         geo.gl.vglRenderer.fallback = function () {return 'canvas';};
     },
 
     /**
+     * Set the displayed image bounds according to the given query string.
+     */
+    _boundsFromQuery: function (query) {
+        if (!this._map || !query) {
+            return;
+        }
+        var bounds = query.split(',').map(function (v) { return +v; });
+        this._map.rotation(bounds[4] * Math.PI / 180);
+        this._map.bounds({
+            left: bounds[0],
+            top: bounds[1],
+            right: bounds[2],
+            bottom: bounds[3]
+        }, null);
```

Comment:
I'm not sure what is going on there, and it will require more time to delve into it that I have right now.  Can we just invert the sign of the vertical parameters in the route (and, obviously invert it again when loading from the route), leaving the `null` parameter for now.